### ISSUE TITLE
System.Windows.Forms.MenuItem is not working

### ISSUE THREAD

**Issue by ghost** [paintdotnet/System.Windows.Forms.Legacy#3]
https://github.com/paintdotnet/System.Windows.Forms.Legacy/blob/3500c3a2846b1280ebc4d7b2a25c4d1b799234a3/System/Windows/Forms/Command.cs#L7

Should use "System.Windows.Forms.Command" in assembly "System.Windows.Forms".

Simple code:

    internal class Command : IDisposable
    {
        private static readonly Type CommandType = typeof(Control).Assembly.GetType(typeof(Command).FullName);
        private static readonly Reflection.PropertyInfo CommandIDProperty = CommandType.GetProperty(nameof(ID), typeof(Int32));
        private static readonly Reflection.MethodInfo CommandDisposeMethod = CommandType.GetMethod(nameof(Dispose), Type.EmptyTypes);

        private readonly Object cmd;

        public Command(ICommandExecutor target) => this.cmd = Activator.CreateInstance(CommandType, target);

        public Int32 ID => (Int32)CommandIDProperty.GetValue(this.cmd);

        public void Dispose() => CommandDisposeMethod.Invoke(this.cmd, null);
    }

**Comment by rickbrew** [comment]
`System.Windows.Forms.Command` in `System.Windows.Forms` is `internal` and cannot be used by this assembly.

I've only gotten this package as far as compiling and working for Paint.NET's scenarios -- that is, getting specific old/legacy plugins to work. 

If something is broken for your project, please describe your scenario and submit a pull request with a fix.

**Comment by ghost** [comment]
Yes, `System.Windows.Forms.Command` in `System.Windows.Forms` is `internal`, but we must use it, because `Control.WmCommand` use it to dispatch `WM_COMMAND` message.

For example,
```
var form = new Form();
form.MouseClick +=
    (sender, e) =>
        new ContextMenu(new[] { new MenuItem("Test", (sender, e) => MessageBox.Show(((MenuItem)sender).Text)) })
            .Show((Control)sender, e.Location);
form.Show();
```
click form, and then click the "Test" menuitem, nothing happen, message box not shown.

#5

**Comment by Unknown025** [comment]
Can confirm, the `Click` event on `MenuItem` does not work otherwise, making context menus unusable.
